packages/coinstac-ui/app/render/components/projects/form-project-controller.js

```diff
@@ -121,13 +155,18 @@ class FormProjectController extends Component {
 
   handleAddMetaFile() {
     app.main.services.files.getMetaFile()
-      .then(metaFile => {
+      .then(metaFilePath => Promise.all([
+        metaFilePath,
+        app.core.projects.getCSV(metaFilePath),
+      ]))
+      .then(([metaFilePath, metaFile]) => {
         this.setState({
           errors: {
             metaFile: null,
           },
           project: {
-            metaFile,
+            metaFile: JSON.parse(metaFile),
```

Comment:
`ProjectService#getCSV` returns the stringified, parsed CSV so we don't have to deal with silly RPC. Something to consider if we have a chance to rework our render/main interactions.